### ISSUE TITLE
Lab: LeNet In TensorFlow, has big bug.

### ISSUE THREAD

**Issue by asimay** [udacity/sdc-issue-reports#624]
in LeNet-Lab.ipynb files:
cross_entropy = tf.nn.softmax_cross_entropy_with_logits(logits, one_hot_y), this is wrong, caused compiled error.
should be:
cross_entropy = tf.nn.softmax_cross_entropy_with_logits(labels=one_hot_y, logits=logits)

**Comment by awbrown90** [comment]
@asimay thanks for the feedback, just made the correction. I think this syntax change came with the newest version of tensorflow.
